composer.json

```
@@ -28,7 +28,7 @@
     "docs": "https://pimcore.com/docs/latest/"
   },
   "require": {
-    "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+    "php": "^8.3",
```

I think you should use `~8.3.0 || ~8.4.0` as you don't know if it'll be compatible with PHP 8.5 yet (as PHP doesn't adhere to SemVer and there may be breaking changes in minor versions).

```suggestion
    "php": "~8.3.0 || ~8.4.0",
```